### ISSUE TITLE
LiveKit in Azure AKS - Need help on configurations for AKS in Helm Charts

### ISSUE THREAD

**Issue by bkbasavaraju** [livekit/livekit-helm#112]
Hi, I am looking for help in deploying LiveKit server in Azure AKS Cluster. Is there any support or sample values.yaml which can be used? Any help would be appreciated.

**Comment by jpflick-stx** [comment]
Were you ever able to get LiveKit server deployed to an Azure AKS Cluster?

**Comment by bkbasavaraju** [comment]
> Were you ever able to get LiveKit server deployed to an Azure AKS Cluster?

@jpflick-stx Nope. I was not able to deploy it through LiveKit helm. I went through STUNner which helped me deploy LiveKit in AKS Cluster.

**Comment by iamMrGaurav** [comment]
@bkbasavaraju were you able to get helm values for livekit in Azure AKS ?

**Comment by mahimairaja** [comment]
@bkbasavaraju did you ?
